### PR TITLE
Test fixed and added to the npm specification

### PR DESCRIPTION
**Pull request by sigv**
The test has been fixed (it looks for the correct library and the PM test does not fail if the config file specifies another nick) and added to the npm specification.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-var irc = require('./client'),
```
Comment (totallymike) [inline test.js:2]: This pull request does more than one thing.  I dispute the need for changing how to look for the library but not the need to fix the tests.

File: test.js [hunk 2]
```diff
@@ -16,23 +16,23 @@ var splitTests = {
...
     'params':  ['action'],
-    'data': 'mike!michael@localhost PRIVMSG tm_test :!test action'
+    'data': 'mike!michael@localhost PRIVMSG tm_test_nick_12345 :!test action'
   }
 };
 
 for (var u in splitTests) {
   var actual = irc.splitcmd(u);
   var expected = splitTests[u];
-
```
Comment (totallymike) [inline test.js:34]: This blank line stays.  It's for readability.

File: test.js [hunk 3]
```diff
@@ -16,23 +16,23 @@ var splitTests = {
...
   assert.deepEqual(actual, expected);
 }
 
 process.exit(0);
+
```
Comment (totallymike) [inline test.js]: This one should go.

File: test.js (hunk 1 above)
Comment (sigv) [inline test.js:2]: It no longer looks for the ./lib/ircnode.js but it needs to be changed anyway, as we moved from ./client to ./lib/ircnode.js.

**Pull request merged**